docs/configs.md

```diff
@@ -0,0 +1,11 @@
+# FAQ
+- We use the partner's organization name in ReportStream as the partner ID
+- Config files are the partner ID plus `.json`
+- Config keys in code are the partner ID
```

Comment:
Could an example of what that would look like be added?

Reply:
Added